hibernate-core/src/main/java/org/hibernate/type/NClobType.java

```diff
@@ -36,7 +39,21 @@ protected boolean registerUnderJavaType() {
 
 	@Override
 	protected NClob getReplacement(NClob original, NClob target, SharedSessionContractImplementor session) {
+		if ( target == null ) {
+			return copyOriginalNClob( original, session );
```

Comment:
Same [comment](https://github.com/hibernate/hibernate-orm/pull/2390/files#r241243365) here.

Reply:
@gsmet, why not just delegate to the LobMergeStrategy?